test/test_Simulation.jl

```diff
@@ -31,17 +31,25 @@ end # testset
 end
 
 
+world2 = World(
+    v0=1.0,
+    n_dogs=100,
+    boxsize=50.0,
+    motion=BrownianMotion(noise=0.5, friction=1.0),
+    sheeps=DenseSheeps(20, n_sheeps=20),
+)
+
 simulation2 = Flase.ClusterTimeSimulation(;
     condition=0,
     dt=0.2,
-    world=world,
+    world=world2,
     plotter=Flase.VoidPlotter()
 )
 
 @testset "ClusterTimeSimulation" begin
     mqd = Flase.MQD(simulation2.world.sheeps)
     msd = Flase.MSD()
-    @show Flase.runsim(simulation2)
     @test Flase.getMSD(msd, simulation2.world.sheeps, simulation2.world) < 0.7
     @test Flase.getMQD(mqd, simulation2.world.sheeps) > 0.1
+    @show Flase.runsim(simulation2)
```

Comment:
why do you run the simulation _after_ the tests?

Reply:
When I did the simulation before, the MSD value wasn't the initial value and therefore wasn't smaller than 0.7. Same for MQD so the tests wouldn't make sense.